scripts/_init.sh

```diff
@@ -84,9 +86,11 @@ done
 [[ "$target" != "Debug" && "$target" != "Release" ]] && \
     echo "target must be either Debug/Release." && usage
 
-echo "Options:"
-echo "  target       = $target"
-echo "PATH="$PATH
+if [[ -z "$GET_VERSION" ]]; then
+    echo "Options:"
+    echo "  target       = $target"
+    echo "PATH="$PATH
+fi
 
-DEPENDENCY_DIR=$DIR/../deps-build/$PLATFORM
+export DEPENDENCY_DIR=$DIR/../deps-build/$PLATFORM/$target
```

Comment:
Added an extra level of path to have both Debug and Release.
In case of Mac, that's the only variant aside from the platform name.